auction.py

```diff
@@ -225,11 +230,12 @@ async def nominate(self, ctx):
   @commands.command()
   async def bid(self, ctx):
     try:
-      client_message = self.auction.bid(ctx.message)
-      # TODO: React with client message
+      self.auction.bid(ctx.message)
+      await ctx.message.add_reaction(self.emojis[0])
```

Comment:
ah, i remember now. `self.auction.bid` can return None, so we have to at least check like... did adding the bid work or did for fail for some reason.